### ISSUE TITLE
I was able to do training IDR but face normals are generated incorrect shape , mesh rendering is not performed

### ISSUE THREAD

**Issue by RanjithKatta** [lioryariv/idr#6]
I have run the code and it nearly took 5:30 hours and after trying with SCAN_ID=24, I have seen there has been this pop up saying that face normal shape incorrect in 2000 epoch,

 **can you give a solution how to resolve it?**

face_normals incorrect shape, ignoring!
----------------------------------------------------------------
RayTracing: object = 1157/2048, secant on 722/741.
----------------------------------------------------------------
dtu_fixed_cameras_24 [2000] (0/49): loss = 0.11349464952945709, rgb_loss = 0.11191627383232117, eikonal_loss = 0.010356109589338303, mask_loss = 5.427672022051411e-06, alpha = 1600.0, lr = 2.5e-05
----------------------------------------------------------------
RayTracing: object = 1359/2048, secant on 634/653.

2. What is the CHECKPOINT is it any integer i.e., 2000 epoch if so , I did run but there was an error saying error  no directory for eval-rendering. can you briefly say what it is? 

what I have done is python evaluation/eval.py  --conf ./confs/dtu_fixed_cameras.conf --scan_id 24 --checkpoint 2000 

resulting face normals incorrect shape for 2 steps after that it notifies face normal is zero for rest of the following and stop executing.

Hoping to know what can be done further.

**Comment by lioryariv** [comment]
Hey, 
"face_normals incorrect shape, ignoring!" is a warning given by trimesh, but it doesn't need to stop the evaluation executing, and the resulted mesh (from the marching cube algorithm) should be exported just fine.

CHECKPOINT is indeed an integer (is the epoch you wish to evaluate), example for running:
python evaluation/eval.py --conf ./confs/dtu_fixed_cameras.conf --scan_id 24 --checkpoint 2000

This will export the reconstructed geometry (ply file) into the folder 'eval' under the experiment name.

I didn't quite understand what was the error you got from eval-rendering.
If you run:
python evaluation/eval.py --conf ./confs/dtu_fixed_cameras.conf --scan_id 24 --checkpoint 2000 --eval_rendering
the evaluation will additionally export the rendering images.

**Comment by RanjithKatta** [comment]
Yeah I got it. There are no issues and no errors regarding above query. Thanks for your reply.
